tests/page-objects/common/about.wdio.page.js

```diff
@@ -0,0 +1,15 @@
+const userName = async () => await (await $('dt=User name')).nextElement();
```

Comment:
I know the about page is unlikely to change but this seems pretty brittle to me.

Reply:
yea, there is zero markup on that page, unfortunately.

Reply:
I wanted to add ... like one assertion to make sure we're on the right page. Any suggestions?

Reply:
I think you could just assert the userName element is present and not worry about the name assertion. Unless we want to add some test-id to make it a bit more reliable on the selector.

Reply:
Ok, cool! Thanks!

Reply:
Done.